index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
+	specification matures, to allow different parts of the web community to iterate
+	on requirements, problems, and solutions. 
+
+	When a [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>. 
+
+	As a specification progresses along the Recommendation Track, Working Groups
+	<em class="rfc2119">should</em> solicit review from some or all of the following
+	stakeholders: 
+
+	<ul>
+		<li>The Working Group responsible for the specification</li>
+		<li>The horizontal review groups (see 6.2.3.2)</li>
+		<li>The Working Groups and organisations documented in the Working Group's charter</li>
+		<li>The general public</li>
+		<li>Other relevant standards organizations</li>
+	</ul>
+
+	The Director <em class="rfc2119">must</em> consider evidence of wide review before
+	approving transitions. Appropriate evidence may include: 
+
+	<ul>
+		<li>Documented requests for wide review</li>
+		<li>Documented discussion and resolution of issues raised during wide review</li>
+	</ul>
+
+<h5 id="horizontal-review">
+Horizontal Review</h5>
+
+	The objective of <dfn id=dfn-horizontal-review export>horizontal review</dfn> is
+	to make sure specifications are robust in terms of accessibility, internationalisation,
+	privacy, security, and technical architecture by the time they become a [=W3C Recommendation=]. 
+
+	Horizontal review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, so that accessibility, internationalisation, privacy,
+	security, and technical architecture are considered throughout. 
+
+	Before a proposed specification transitions onto the Recommendation Track, the
+	group responsible for the specification <em class="rfc2119">must</em> solicit a
+	horizontal review from each of the horizontal review groups (in many cases,
+	submitting a self-review to the reviewing group will satisfy this requirement).
+
+	The <dfn id=dfn-horizontal-review-groups export>Horizontal Review Groups</dfn> are:
+
+	<ul>
+		<li><a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures Working Group</a></li>
+		<li><a href="https://www.w3.org/International/core/Overview">Internationalisation Working Group</a></li>
+		<li><a href="https://www.w3.org/Privacy/">Privacy Interest Group</a></li>
+		<li><a href="https://www.w3.org/2001/tag/">Technical Architecture Group</a></li>
+	</ul>
+
+	When the [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to
+	<a href="https://lists.w3.org/Archives/Public/horizontal-review-announce/">horizontal-review-announce@w3.org</a>. 
+
+	Horizontal review <em class="rfc2119">must</em> be solicited from the Horizontal
+	Review groups at least 60 days before requesting transition to [=Candidate Recommendation=]. 
```

Comment:
This is a big problem, because it puts a 2 month brake on all transitions. We should not be adding new delays into the process. This delay is present with this proposed wording even if all the Horizontal Reviews are completed earlier. If what is intended is that HR groups get at least 60 days to review all specifications, then it could be reworded as:
>Working Groups must not request transition to Candidate Recommendation either until at least 60 days have passed before Horizontal Review has been solicited or until all Horizontal Reviews have been completed, whichever is the earlier.

I'm sure someone could word it even better while keeping the same sense.

A different/additional option is to move this requirement from being stated here like this to being a permitted evidence response when requesting transition to CR, i.e. a WG would be allowed to cite "no response" and provide evidence that they sent the HR request >60 days before the transition request. I've included this in a proposal for a different list of allowed evidence below.

Reply:
I don't understand the first part of your "either"?

I can see the logic in the second part, but I'm not sure it entirely resolves the whole problem. namely that WG could cajole HR groups into responding to a timetable they can't meet, then using the lack of HR response as an easy pass on heading to transition without it.

Reply:
> WG could cajole HR groups into responding to a timetable they can't meet, then using the lack of HR response as an easy pass on heading to transition without it.

Ah, right, thanks, there's a point I missed, which is that an HR review may have _begun_ within 60 days but not yet be _completed_, (or in a "we went as far as we could" state). The more I think about it the more my second option of dropping this requirement and just adding it into the requirements for transition request is an easier fix, because my proposed list doesn't include "HR review started but we're still working on getting to resolution", and it's clearer.

Reply:
I've updated the text based on @dwsinger's suggestion:
>Transition to Candidate Recommendation must not be requested unless either the Horizontal Review Groups have been given at least 60 days notice to review the specification or the Horizontal Review Groups have completed their reviews.>